process-compose.yaml

```diff
@@ -429,3 +430,11 @@ processes:
       # See https://github.com/docker-library/postgres/issues/146 for discussion.
       success_threshold: 2
       failure_threshold: 20
+
+  block-explorer:
+    command: docker run --rm -p 3000:3000 -e QUERY_SERVICE_URI --platform=linux/amd64 ghcr.io/espressosystems/espresso-block-explorer:release-cappuccino
```

Comment:
Is the `--platform` flag needed? And does this work on ARM based macs?

Reply:
At the moment this does not, but it is something I have brought up with @Ancient123.  It was disabled due to the build timing out.

Reply:
This should now be resolved in commit 25bca33758a88b628e6716de43be8e8c9b52e7fb.